src/encoded/static/components/browse/MicroscopySearchView.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+import React from 'react';
+import memoize from 'memoize-one';
+import _ from 'underscore';
+import url from 'url';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['url' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)